src/notation/internal/notationinteraction.cpp

```diff
@@ -1240,6 +1242,17 @@ bool NotationInteraction::drop(const PointF& pos, Qt::KeyboardModifiers modifier
         accepted = true;
     }
     break;
+    case ElementType::SLUR:
+    {
+        EngravingItem* el = dropTarget(m_dropData.ed);
```

Comment:
I'm curious how this worked before (for example, in MU3)... Because there may be a more general problem (for a lot of elements, not just slurs)

Reply:
I looked into it and the MU3 code is completely different as far as I can tell. It looks like this code was written for MU4, and I wonder if dragging and dropping slurs worked at all since the move.

This problem could definitely affect other things but I made sure to test every spanner in the palette and they all seemed to work fine! If other problems are found we know where to look for the solution at least.